### DISCUSSION TITLE
文章滚动时顶部导航栏未触发应有的变色、变标题效果

### DISCUSSION THREAD

**Discussion by kahvia-d** [anzhiyu-c/hexo-theme-anzhiyu#253] [Q&A]
[问题博客](https://blog.kahvia.cn/2024/06/21/latex)：blog.kahvia.cn
我在用安知鱼主题的时候，发现点开一篇文章后，下滑滚动的时候，顶部导航栏不会发生改变。我看鱼佬的博客是会变背景色，然后显示文章名称的，并且点击可以返回顶部。
我这个控制台有报错`main.js:1737  Uncaught ReferenceError: GLOBAL_CONFIG_SITE is not defined`，但是我不确定问题具体出在哪里。鱼佬的博客在滚动的时候，header标签会自动添加nav-fixed这个class，但是我那个不会自动添加这个。我在控制台手动添加后，本该显示文章标题的地方显示了“PAGE_NAME”。
方便给我一点修改意见吗？

**Comment by kahvia-d** [comment]
已经解决了。问题出在hexo网站配置文件_config.yml里面，`title: Kahvia's Home`，这个标题在初始化GLOBAL_CONFIG_SITE变量时，出现了'Kahvia's Home'的情况，使得前后单引号匹配出问题。所以我将其改成了KahviaのHome就恢复正常了。

**Reply by zippedalo** [reply]
其实可以用反斜杠转义一下的

**Comment by anzhiyu-c** [comment]
也可以使用双引号包裹
